### ISSUE TITLE
Ignore Specific Updates

### ISSUE THREAD

**Issue by The-Fireplace** [ATLauncher/ATLauncher#469]
**Is your feature request related to a problem? Please describe.**
It is annoying when updates I am purposely not installing (whether because it's an alpha I know will break things, or because it's a file for the other mod loader that the author failed to mark) are shown for some mods.

**Describe the solution you'd like**
An "Ignore this update" button that lets me ignore the currently suggested update. Updates would show again when a new file is added that can be used with the current instance.

**Comment by CorneliusCornbread** [comment]
Agreed, just had an issue with dependency hell in a modpack due to an update to my mods updating a dependency to an alpha version. Which caused problems amongst the other mods.
